src/main/java/com/example/petbuddybackend/service/care/state/CareStateMachine.java

```diff
@@ -77,9 +104,13 @@ private void initCaretakerTransitions(TransitionManager transitionManager) {
 
     private void initGlobalTransitions(TransitionManager transitionManager) {
```

Comment:
nwm czy musimy dodawac tutaj te transakcje, bo ich i tak chyba w kodzie nie wykorzystujemy, bo robimy to przez query, chyba, ze gdzies wykorzystujemy?

Reply:
Całości bym nie usuwał, ale mogę ten init wywalić plus `transitionBothRoles` żeby nie zostawiać nieużywanego kodu. Ale `TransitionManager` już bym zostawił tak jak jest. Może kiedyś się przyda ta dodatkowa operacja